<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DynamicsDialog</class>
 <widget class="QDialog" name="DynamicsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Collect Data Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLineEdit" name="totalTimeEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="iterationTimeLabel">
     <property name="text">
      <string>Total measurment time in sec.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="totalTimeLabel">
     <property name="text">
      <string>Time between measurements in sec.</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="iterationTimeEdit">
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>iterationTimeEdit</tabstop>
  <tabstop>totalTimeEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DynamicsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DynamicsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
